src/App.js

```diff
@@ -1,59 +1,67 @@
-import React from 'react';
-import { Routes, Route, NavLink } from 'react-router-dom';
+import { Routes, Route, useNavigate } from 'react-router-dom';
 import './App.css';
-import Firebase from './Pages/firebase';
+import { getToken } from '@the-collab-lab/shopping-list-utils';
+import { useState, useEffect } from 'react';
+import AddItem from './components/AddItem/AddItem';
+import NavLinks from './components/Navigation/NavLinks';
+import ItemList from './pages/ItemList';
 
-function List() {
-  return (
-    <>
-      <h1>List of Items</h1>
-    </>
-  );
-}
+function App() {
+  const [token, setToken] = useState(null);
+  const navigate = useNavigate();
 
-function AddAnItem() {
-  return (
-    <>
-      <h1>Add an Item</h1>
-    </>
-  );
-}
+  useEffect(() => {
+    const json = localStorage.getItem('dummy');
+    const loadedToken = JSON.parse(json);
+    if (loadedToken) {
+      navigate('/item-list');
+      setToken(loadedToken);
+    }
+  }, []);
 
-function NavLinks() {
-  return (
-    <nav style={{ position: 'fixed', bottom: 0 }}>
-      <NavLink
-        to="/"
-        style={({ isActive }) =>
-          isActive ? { fontWeight: 'bold' } : { fontWeight: 'normal' }
-        }
-      >
-        List of Items
-      </NavLink>
-      <NavLink
-        to="/add-an-item"
-        style={({ isActive }) =>
-          isActive ? { fontWeight: 'bold' } : { fontWeight: 'normal' }
-        }
-      >
-        Add an Item
-      </NavLink>
-    </nav>
-  );
-}
+  useEffect(() => {
+    const json = JSON.stringify(token);
+    localStorage.setItem('dummy', json);
+  }, [token]);
+
+  const onClick = () => {
+    localStorage.setItem('dummy', JSON.stringify({ token: getToken() }));
```

Comment:
This is saving the token as a stringified object. 

<img src="https://cdn.zappy.app/f29e505af15cfc81a8df910bade12855.png" />

Instead, I think that we want to save it as just a string, which would be `localStorage.setItem('token', JSON.stringify(getToken())`